compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/pipeline_tile_and_fuse.mlir

```diff
@@ -630,8 +630,8 @@ hal.executable public @main {
 // CHECK-DAG:      memref.alloc() : memref<64x72xi8, #gpu.address_space<workgroup>>
 // CHECK-DAG:      memref.alloc() : memref<64x136xi8, #gpu.address_space<workgroup>>
 // CHECK-COUNT-32: amdgpu.wmma {{.*}} : vector<16xi8>, vector<16xi8>, vector<8xi32>
-// CHECK:          vector.transfer_write {{.*}} : vector<4x1x2x8x1xi8>, memref<16x16x196x8x2xi8, #hal.descriptor_type<storage_buffer>>
-// CHECK:          vector.transfer_write {{.*}} : vector<2x8x1x4x1xi8>, memref<196x8x2x16x16xi8, #hal.descriptor_type<storage_buffer>>
+// CHECK:          vector.transfer_write {{.*}} : vector<4x1x2x8x1xi8>, memref<8x16x4x8x2xi8, strided<[50176, 3136, 16, 2, 1], offset: ?>, #hal.descriptor_type<storage_buffer>>
+// CHECK:          vector.transfer_write {{.*}} : vector<2x8x1x4x1xi8>, memref<4x8x2x8x16xi8, strided<[4096, 512, 256, 16, 1], offset: ?>, #hal.descriptor_type<storage_buffer>>
```

Comment:
Do you know what causes this pipeline test to change? I'd expect the result of the pipeline tests to mostly stay the same.